src/__tests__/use-document/__fixtures__/custom-document-with-gip/pages/page.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 
 export default function CustomDocumentWithGIP_Page() {
-  return <>/custom-document-with-gip/page</>;
+  return <span>/custom-document-with-gip/page</span>;
```

Comment:
Using `<span>` otherwise `expect(actual).toEqual(expected);` would fail due to `renderToStatic*` behaviour which will somehow truncate strings -- this would not affect actual page looks so it's not really an issue.